### ISSUE TITLE
Checksum for windows installer not matching

### ISSUE THREAD

**Issue by b-f-** [paritytech/parity#7208]
On https://www.parity.io/
Website checksum for windows installer
![image](https://user-images.githubusercontent.com/2358754/33629662-228feb12-da05-11e7-9938-16eb54a7559f.png)

Local checksum
![image](https://user-images.githubusercontent.com/2358754/33629668-2d448608-da05-11e7-8654-ba700879132e.png)

**Comment by kirushik** [comment]
`a5f11c2b9a51246924fba22aa6d14762` is a valid MD5 hash for the file I've just downloaded from https://d1h4xl4cr1h0mo.cloudfront.net/stable/x86_64-pc-windows-msvc/InstallParity.exe (download link on the official site).

The same file has a SHA256 hash `4f3cd4d3cab9bd043675bd833cf1b47f0396e7c544707874fb7f88177ebcfa81` which matches the one on your screenshot — so you've downloaded the correct and valid installer.

We should really improve, and explicitly state what checksum algorithm should you use to verify right there on the website; I've raised this issue with a website team.
